src/elm/external.js

```diff
@@ -31,6 +31,11 @@ class Retrieve extends Expression {
       records = records.filter(r => range.includes(r.getDateOrInterval(this.dateProperty)));
     }
 
+    if (Array.isArray(records)) {
+      records.forEach(rec => ctx.evaluatedRecords.push(rec));
```

Comment:
Yeah good call. You sure you don't want the following though? ;)

``` JavaScript
ctx.evaluatedRecords.push(Array.isArray(records) ? ...records : records);
```

Reply:
I actually started to comment that you _could_ do it that way, but then I second-guessed myself because I wasn't sure if a spread operator would work inside a ternary like that.  Does it?

I'm not one of those ternary operator haters.  Are you?